src/backend/base/langflow/components/vectorstores/chroma.py

```
@@ -157,3 +183,60 @@ def _add_documents_to_vector_store(self, vector_store: "Chroma") -> None:
+
+        query = self.search_query
+        if not query:
+            self.status = ""
+            return []
+
+        mode = self.search_type
+        k = self.number_of_results
+        filt = self.advance_search_filter
+        threshold = float(self.sim_threshold or 0.0)
+
+        if mode == "Similarity with Score" and hasattr(vs, "similarity_search_with_score"):
```

Flake8 (7.2.0)

[error] line too long (91 > 79 characters)

(E501)